workflows/microbiome/taxonomic-binninge-evaluation/FAIRyMAGs-taxonomic-binning-evaluation-tests.yml

```
@@ -0,0 +1,53 @@
+- doc: Test outline for FAIRyMAGs-taxonomic-binning-evaluation
+  job:
+    Biobox File(s):
+      class: Collection
+      collection_type: list
+      elements:
+      - class: File
+        identifier: marine_pooled_gsb_trimmed
+        location: https://zenodo.org/records/16621848/files/Biobox%20File(s)_marine_pooled_gsb_trimmed.tabular
+    Assembly(s):
+      class: Collection
+      collection_type: list
+      elements:
+      - class: File
+        identifier: marine_pooled_gsa_trimmed
+        location: https://zenodo.org/records/16621848/files/Assembly(s)_marine_pooled_gsa_trimmed.fasta
+    Reads (paired into forward and reverse):
```

The label here is redundant with the collection type
```suggestion
    Reads:
```